x-pack/plugin/wildcard/src/test/java/org/elasticsearch/xpack/wildcard/mapper/WildcardFieldMapperTests.java

```diff
@@ -912,4 +917,42 @@ private String getRandomWildcardPattern() {
         }
         return sb.toString();
     }
+
+    /**
```

Comment:
This one doesn't extend from `ESSingleNodeTestCase`! We probably should stop all of them from extending from it, but that is a job for another day.